lib/live_debugger/app/debugger/components_tree/web/components_tree_live.ex

```diff
@@ -166,57 +147,49 @@ defmodule LiveDebugger.App.Debugger.ComponentsTree.Web.ComponentsTreeLive do
       ) do
     socket
     |> assign(highlight_disabled?: true)
```

Comment:
Is this assign used anywhere? I think you removed it in before CR, and reverted after applying CR suggestions